src/examples/form/form.tsx

```diff
@@ -1,60 +1,116 @@
+                    title: 'Red',
+                },
+                {
+                    type: 'string',
+                    const: 'yellow',
+                    title: 'Yellow',
+                },
+                {
+                    type: 'string',
+                    const: 'green',
+                    title: 'Green',
+                },
+                {
+                    type: 'string',
+                    const: 'blue',
+                    title: 'Blue',
+                },
+                {
+                    type: 'string',
+                    const: 'black',
+                    title: 'Black',
+                },
+            ],
         },
-        colorMask: {
+        sidekicks: {
             type: 'array',
-            title: 'Color mask',
-            description: 'Pick a color',
+            title: 'Sidekicks',
+            description: 'Please select your sidekicks',
             uniqueItems: true,
             items: {
-                title: 'Color',
+                title: 'Friends',
                 type: 'string',
                 anyOf: [
                     {
                         type: 'string',
-                        enum: ['#ff0000'],
-                        title: 'Red',
+                        const: 'super',
+                        title: 'Superman',
+                    },
+                    {
+                        type: 'string',
+                        const: 'widow',
+                        title: 'Black Widow',
+                    },
+                    {
+                        type: 'string',
+                        const: 'america',
+                        title: 'Captain America',
+                    },
+                    {
+                        type: 'string',
+                        const: 'squirrel',
+                        title: 'Squirrel Girl',
                     },
                     {
                         type: 'string',
-                        enum: ['#00ff00'],
-                        title: 'Green',
+                        const: 'devil',
+                        title: 'Daredevil',
                     },
                     {
                         type: 'string',
-                        enum: ['#0000ff'],
-                        title: 'Blue',
+                        const: 'marvel',
+                        title: 'Captain Marvel',
+                    },
+                    {
+                        type: 'string',
+                        const: 'fantastic',
+                        title: 'Mr. Fantastic',
                     },
                 ],
             },
         },
-        date: {
-            type: 'string',
-            title: 'Date',
-            format: 'date',
-            description: 'Select a nice date',
+        newsletters: {
+            type: 'boolean',
+            title: 'I want to receive daily newsletters!',
+            default: false,
+            const: true,
```

Comment:
> `const: true`

This makes me a bit confused… what does this actually indicate? That the value is `true` when this is checked? Or that the value has to be `true` for it to validate? I don't quite understand how this relates to what I thought `const` meant on the options above…

Reply:
`const: true` means that the value of the field must equal to `true` in order to be valid. From the example with the options above, all the options are part of a `oneOf`, so the field there will be valid if the field is equal to any of those values. Luckily, you would normally not write the JSON schema by hand :dizzy_face:

Reply:
Ah, so that's what the `anyOf` thing means…

How come the option field is still valid if no option is selected? Because it's not required?

Reply:
Yeah, it's not required :slightly_smiling_face: